packages/execute-driver-plugin/lib/plugin.js

```diff
@@ -24,14 +24,15 @@ export default class ExecuteDriverPlugin extends BasePlugin {
    * already been attached to the currently running session.
    *
    * @param {function} next - standard behaviour for executeDriverScript
-   * @param {BaseDriver} driver - Appium driver handling this command
+   * @param {import('@appium/base-driver').BaseDriver} driver - Appium driver handling this command
```

Comment:
The previous type was incorrect

```suggestion
   * @param {import('@appium/types').ExternalDriver} driver - Appium driver handling this command
```

Reply:
If that doesn't compile, then

```suggestion
   * @template {import('@appium/types').Constraints} [C={}]
   * @param {import('@appium/types').ExternalDriver<C>} driver - Appium driver handling this command
```